packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -1171,4 +1171,278 @@ describeSpec('Limbo Documents:', [], () => {
       );
     }
   );
+
+  specTest(
+    'Fix #8474 - Limbo resolution for documentC is removed even if documentC updates occurred earlier in the global snapshot window',
```

Comment:
nit: Can you modify this test case's name to be less specific (e.g. avoid mentioning specific document names).

Reply:
I've cleaned up the test name and some of the step names to make the test more readable.